src/components/App/App.tsx

```diff
@@ -30,6 +30,7 @@ const App = () => {
       boardState.id = newPuzzleNum;
       boardState.puzzle = structuredClone(puzzles[newPuzzleNum]);
       document.location.hash = newPuzzleNum.toString();
+      localStorage.setItem('puzzleNum', newPuzzleNum.toString());
```

Comment:
I saw that there is a useLocalStorage hook maybe you should use it.

Reply:
> I saw that there is a useLocalStorage hook maybe you should use it.

Exactly, it makes sense to use it.
Or, a least the `StorageEntry` class, which is inside the useLocalStorage hook file.

Reply:
Thanks for the feedback. New to React hooks and stuff :)
Will try to fix this later and update.